tests/sys/posixshm/posixshm_test.c

```diff
@@ -1718,16 +1718,16 @@ ATF_TC_BODY(largepage_mprotect, tc)
 		memset(addr, 0, ps[i]);
 
 		/* Map two contiguous large pages and merge map entries. */
-		addr1 = mmap(addr + ps[i], ps[i], PROT_READ | PROT_WRITE,
-		    MAP_SHARED | MAP_FIXED | MAP_EXCL, fd, 0);
```

Comment:
Hrm, how did this work before? We're replacing a mapping here, so MAP_EXCL should have caused a failure...

Reply:
Is this test passing upstream? I stared at the code for a while and it seems it should not.

Reply:
Previously the reservation was unmapped. In a single-threaded, non-quarantining environment that is sufficient to find unused address space, but we break that assumption and currently have no easy way to force the mapping out of quarantine (that argues for implemented the bitmap based revocation system @nwf actually designed rather than my single merged entry approach). Dropping the unmap entirely just works though.

Reply:
Ah yes, that makes sense now. Thanks